lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -2826,6 +2830,10 @@ IRBuilderAsmJs::BuildFloat2(Js::OpCodeAsmJs newOpcode, uint32 offset, Js::RegSlo
     case Js::OpCodeAsmJs::I_Conv_VTF:
         instr = IR::Instr::New(Js::OpCode::Ld_A, dstOpnd, srcOpnd, m_func);
         break;
+    case Js::OpCodeAsmJs::Copysign_Flt:
+    case Js::OpCodeAsmJs::Trunc_Flt:
+    case Js::OpCodeAsmJs::Nearest_Flt:
```

Comment:
can you add an Assert(UNREACHED) here? I'm pretty sure this will crash later with nullptr dereference if we come here, but would be good to have more explicit error if trying to jit these that they are not yet implemented